packages/web-components/src/components/masthead/cloud/cloud-masthead-composite.ts

```diff
@@ -212,7 +219,11 @@ class DDSCloudMastheadComposite extends DDSMastheadComposite {
                       `
                   )}
                 </dds-cloud-masthead-profile>
-                <dds-cloud-button-cta kind="ghost">${contactUsButton?.title}</dds-cloud-button-cta>
+                ${hasContact
+                  ? html`
+                      <dds-cloud-button-cta kind="ghost">${contactUsButton?.title}</dds-cloud-button-cta>
```

Comment:
We'll need to account for the `data-ibm-contact=` attribute to trigger the "Let's Chat" contact module. Here's the link to the doc that Jeff shared with us - https://www.ibm.com/standards/web/v19a-archive/components/contact-module/#call-to-actions